index.html

```diff
@@ -71,43 +71,72 @@ <h2>A project for first-time contributions</h2>
+            </p>
+            <p class="about">about you</p>
+            <div class="resources">
+              <p>3 Useful Dev Resources</p>
+              <ul>
+                <li>
+                  <a href="#" target="_blank" title="First Resource">
+                    Resource 1
+                  </a>
+                </li>
+                <li>
+                  <a href="#" target="_blank" title="Second Resource">
+                    Resource 2
+                  </a>
+                </li>
+                <li>
+                  <a href="#" target="_blank" title="Third resource">
+                    Resource 3
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <!-- ________*END TEMPLATE* Contributor card END ________  -->
+
+        <!-- COPY everything ABOVE this, from contributor card start to end along with the "START" and "END" comment lines -->
+
+        <!-- ============== ^^^^ TEMPLATE ^^^^ ============== -->
+        <!-- ============================================================== -->
+        <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
         <div class="card">
-          <p class="name">Your name</p>
+          <p class="name">Terrish Thomas</p>
           <p class="contact">
-            <i class="fab fa-twitter"></i>
-            <a href="www.twitter.com" target="_blank">Your handle</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/Terrish73" target="_blank">@Terrish73</a>
           </p>
-          <p class="about">about you</p>
+          <p class="about">I'm a BCA graduate with basic knowledge in coding and design.I'm learning python.</p>
           <div class="resources">
             <p>3 Useful Dev Resources</p>
             <ul>
               <li>
-                <a href="#" target="_blank" title="First Resource">
-                  Resource 1
+                <a href="https://stackoverflow.com/" target="_blank" title="First Resource">
+                  Good site to ask questions
                 </a>
               </li>
               <li>
-                <a href="#" target="_blank" title="Second Resource">
-                  Resource 2
+                <a href="https://www.freecodecamp.org/" target="_blank" title="Second Resource">
+                  A very good site to learn coding
                 </a>
               </li>
               <li>
-                <a href="#" target="_blank" title="Third resource">
-                  Resource 3
+                <a href="https://www.w3schools.com/" target="_blank" title="Third resource">
+                  W3 schools
                 </a>
               </li>
             </ul>
           </div>
         </div>
         <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
change `END TEMPLATE` with your username
add a line break after the end of you card